### ISSUE TITLE
markdown code fences - mariadb language as alias to mysql

### ISSUE THREAD

**Issue by hrvoj3e** [go-gitea/gitea#18254]
### Gitea Version

1.16.0

### Operating System

Linux

### Browser Version

Chrome 96 on Linux

### Can you reproduce the bug on the Gitea demo site?

Yes

### Description

 - https://try.gitea.io/lfaoanl/test/issues/1
 - https://github.com/alecthomas/chroma/issues/590

```mysql
SELECT z.label FROM mytable AS z WHERE id=1;
```

```mariadb
-- should highlight the same as mysql above
SELECT z.label FROM mytable AS z WHERE id=1;
```

Html that renders from Gitea
```html
<pre><code class="chroma language-mariadb">
SELECT z.label FROM mytable AS z WHERE id=1;
</code></pre>
```

### Screenshots

![image](https://user-images.githubusercontent.com/4988133/149167208-c61ac21a-f541-4310-9104-a0da9271fa48.png)

**Comment by hrvoj3e** [comment]
Upstream solved this. Please pull new version for next release.

**Comment by wxiaoguang** [comment]
We have a PR for upgrading Chroma.

* https://github.com/go-gitea/gitea/pull/18270

Let's follow it.

**Comment by hrvoj3e** [comment]
It seems they have not included this in 0.10.0 release. Don't know why.

https://github.com/alecthomas/chroma/blob/v0.10.0/lexers/m/mysql.go

**Comment by wxiaoguang** [comment]
That changes hasn't been release in Chroma. As disscussed in #18270 , we merge 0.10.0 now, and once Chroma has a new release, we will update again then.

**Comment by lunny** [comment]
Put this to v1.18.0 but put back once Chroma released v2.0.0
